pkg/sign/signer.go

```
@@ -33,20 +51,26 @@ func Bundle(content Content, keypair Keypair, fulcio *Fulcio, idToken string, ti
 	content.Bundle(bundle, signature, digest, keypair.GetHashAlgorithm())
 
 	// Add verification information to bundle
-	if fulcio != nil {
-		certBytes, err := fulcio.GetCertificate(keypair, idToken)
+	var pubKeyPEM []byte
+	if opts.Fulcio != nil && opts.IDToken != "" {
```

Same here, can we return an error if IDToken is not set?